framework/console/Controller.php

```diff
@@ -102,7 +102,7 @@ public function runAction($id, $params = [])
                 if (in_array($name, $options, true)) {
                     $default = $this->$name;
                     if (is_array($default)) {
-                        $this->$name = preg_split('/(?!\(\d+)\s*,\s*(?!\d+\))/', $value);
+                        $this->$name = preg_split('/(?!\((\' | ")*[^"\']+)\s*,\s*((?![^"\'(\)]*(\'|")*\)))/', $value);
```

Comment:
i am no good reading regex, what does this line is do? how is it different than before?